spring-boot-project/spring-boot/src/main/java/org/springframework/boot/task/TaskExecutorBuilder.java

```diff
@@ -289,12 +288,8 @@ public ThreadPoolTaskExecutor build() {
 		map.from(() -> this.threadNamePrefix).whenHasText()
 				.to(taskExecutor::setThreadNamePrefix);
 		map.from(() -> this.taskDecorator).to(taskExecutor::setTaskDecorator);
-
-		if (!CollectionUtils.isEmpty(this.taskExecutorCustomizers)) {
-			for (TaskExecutorCustomizer customizer : this.taskExecutorCustomizers) {
-				customizer.customize(taskExecutor);
-			}
-		}
+		map.from(() -> this.taskExecutorCustomizers).to((customizers) -> customizers
```

Comment:
I am not super fan of this construct (you're not really mapping anything on the target object).

Reply:
Thanks for the quick feedback! I reverted this change in 12ee725.